iphone/Resources/app.js

```diff
@@ -6,17 +6,44 @@
  * to trigger a log that is displayed in the Xcode console.
  */
 
+'use strict';
+
 var win = Ti.UI.createWindow({
-    backgroundColor: '#fff'
+	backgroundColor: '#fff'
 });
 
 var btn = Ti.UI.createButton({
-    title: 'Trigger'
+	title: 'Trigger'
 });
 
-btn.addEventListener('click', function() {
-    Ti.API.info('Hello world!');
+btn.addEventListener('click', function () {
+    var win2 = Ti.UI.createWindow({
+        backgroundColor: '#fff'
+    });
+    win2.addEventListener('close', function () {
+        console.log('CLOSE  WINDOW 2');
+    });
```

Comment:
<!--
  1 failure:  iphone/Resources/...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: iphone/Resources/app.js;
  Line: 25;
-->
- :no_entry_sign: iphone/Resources/app.js line 25 – Expected indentation of 1 tab but found 4 spaces. (indent)